src/core/hle/service/am/am.cpp

```diff
@@ -418,6 +429,94 @@ InstallStatus InstallCIA(const std::string& path,
     return InstallStatus::ErrorInvalid;
 }
 
+InstallStatus InstallFromNus(u64 title_id, int version) {
+#ifdef ENABLE_WEB_SERVICE
+    LOG_DEBUG(Service_AM, "Downloading {:X}", title_id);
+
+    Service::AM::CIAFile installFile(Service::AM::GetTitleMediaType(title_id));
+    std::string path = fmt::format("/ccs/download/{:016X}/tmd", title_id);
+    if (version != -1)
+        path += fmt::format(".{}", version);
+    auto tmdResponse = WebService::NUS::Download(path);
+    if (!tmdResponse) {
+        LOG_ERROR(Service_AM, "Failed to download tmd for {:016X}", title_id);
+        return InstallStatus::ErrorFileNotFound;
+    }
+    FileSys::TitleMetadata tmd;
+    tmd.Load(*tmdResponse);
+
+    path = fmt::format("/ccs/download/{:016X}/cetk", title_id);
+    auto cetkResponse = WebService::NUS::Download(path);
```

Comment:
```suggestion
    auto cetk_response = WebService::NUS::Download(path);
```
Dito with all other occurences below